src/components/App.js

```diff
@@ -0,0 +1,15 @@
+import '../styles/App.css';
+import { Routes, Route } from 'react-router-dom';
+import Home from '../routes/Home';
+import CurrencyDescription from '../routes/CurrencyDescription';
+
+function App() {
```

Comment:
- [ ] As required for the project, kindly ensure that the project files are well organized. It is okay if all components (stateless and stateful components) are placed in the components folder. Right now most of your **components files(stateless and stateful components) are out of the `components` directory**,... Below is a suggestion example of how your files have to be structure(**Other options are also ok as long as code is organized in a logical way.**)
```
app-root-directory
  |- package.json
  |- README.md
  |- /public
    |- index.html
  |- /src
    |- app.css
    |- index.js
    |- /components
    |- /container
    |- /pages
    |- /redux
  |- /node_modules
```